SECURITY.md

```diff
@@ -0,0 +1,57 @@
+# Security Policy
+
+Quiet is an experimental, peer-to-peer, end-to-end encrypted, and serverless team chat application built on Tor & IPFS. While we aim to improve user autonomy and privacy beyond what centralized messengers can offer, **Quiet is not audited** and **should not be relied upon in high-risk scenarios requiring proven, thoroughly vetted security guarantees**.
+
+## Supported Versions and Maintenance
+
+Only the latest release of Quiet is officially supported with security fixes and updates. We do not maintain long-term support (LTS) branches. When security issues are discovered, we will provide timely patches and point releases for the most recent version.
+
+| Version | Supported          |
+|---------|--------------------|
+| Latest  | ✔ Active support (timely patches & point releases) |
+| Older   | ❌ No formal support|
+
+## Threat Model and Metadata Exposure
+
+We have published a [Threat Model](https://github.com/TryQuiet/quiet/wiki/Threat-Model) detailing our assumptions, adversaries, desired security invariants, and known weaknesses. Since Quiet is under active development, some security features (e.g., deletion, user removal, direct messages, private channels) are still evolving.
+
+### iOS Push Notifications
+
+If enabled, iOS push notifications rely on a centralized Apple service. While message content remains encrypted, push notifications may reveal timing metadata to Apple’s servers. This could potentially allow a sophisticated adversary to learn when a user is active or receiving messages.
+
+- **Optional Notifications:** Users who prioritize privacy may opt out of push notifications. Disabling notifications prevents this timing metadata from being exposed, at the cost of not receiving immediate alerts for new messages.
+
+We will provide more detailed descriptions of metadata exposure in updated threat model documentation.
```

Comment:
None of this is implemented yet. I think this belongs in the threat model doc and we should only update it once the way notifications work changes. I'd like to consolidate this info in the threat model doc because having information split between the two docs could lead to misunderstandings or mistakes. Thoughts?